sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -220,17 +220,17 @@ getOrBuild(KernelProgramCache &KPCache, KeyT &&CacheKey, AcquireFT &&Acquire,
 
 static bool isDeviceBinaryTypeSupported(const context &C,
                                         RT::PiDeviceBinaryType Format) {
+  // All formats except PI_DEVICE_BINARY_TYPE_SPIRV are supported.
```

Comment:
I think that decision whether a binary type is supported by a particular context or not should be made by plugin (or BE) decision associated with the context. Looks like we need a new PI function. Can you please add a TODO?